File: Data Structures & Algorithms/find-target-in-rotated-sorted-array/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number}
     */
    search(nums, target) {
        let result = -1

        for (let i = 0; i < nums.length; i++) {
            if (nums[i] === target) {
                return i
            }
        }

        return result
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d = {}
        for i in range(len(nums)):
            h = target - nums[i]
            if h in d:
                return [d[h], i]

            d[nums[i]] = i
        
        return []

